tasks/install.yml

```diff
@@ -8,6 +8,7 @@
   retries: 5
   delay: 2
   with_items: "{{ node_exporter_dependencies }}"
+  ignore_errors: yes
```

Comment:
If the lib for selinux isn't installed, Ansible won't be able to enable the port in the case where selinux is enabled. I see that this PR ignores errors on both the install and the port enable. In that scenario where internet connectivity isn't present, won't this result in a failure to configure node exporter properly -- and the role won't throw the error properly?  Perhaps your integration test would catch the failure. Thoughts?

I dislike having to run a mirror for pip, but maybe that's necessary.

Reply:
For now, we can explicitly mention that the default port `9100` for exporting the metrics is not opened on `selinux` enabled systems. I will create an issue/bug for this and work on providing the packages statically on a file server. Can this PR be merged with this exception explicitly mentioned in the documentation and an open bug to fix it?

Reply:
This has now been taken care in #2